x/stake/keeper/validator.go

```diff
@@ -281,42 +304,34 @@ func (k Keeper) UpdateValidator(ctx sdk.Context, validator types.Validator) type
 // The correct subset is retrieved by iterating through an index of the
 // validators sorted by power, stored using the ValidatorsByPowerIndexKey.
 // Simultaneously the current validator records are updated in store with the
-// ValidatorsBondedIndexKey. This store is used to determine if a validator is a
-// validator without needing to iterate over the subspace as we do in
+// ValidatorsBondedIndexKey. This store is used to determine if a validator is
+// a validator without needing to iterate over the subspace as we do in
 // GetValidators.
 //
-// Optionally also return the validator from a retrieve address if the validator has been bonded
+// Optionally also return the validator from a retrieve address if the
+// validator has been bonded
+//
 // nolint: gocyclo
 // TODO: Remove the above golint
 func (k Keeper) UpdateBondedValidators(ctx sdk.Context,
 	affectedValidator types.Validator) (updatedVal types.Validator) {
```

Comment:
I think normally named returned parameters are best suited for short and concise methods/functions. A method such as this, I think it may be more clear if we explicitly return a given variable.

Reply:
yeah agreed, return the variable explicitly now, also added a bool to make it more clear when something should be read from the return